code/modules/mob/living/carbon/xenomorph/XenoAttacks.dm

```diff
@@ -90,7 +90,7 @@
 //Hot hot Aliens on Aliens action.
 //Actually just used for eating people.
 /mob/living/carbon/xenomorph/attack_alien(mob/living/carbon/xenomorph/M)
-	if (M.fortify || M.burrow)
+	if (M.fortify || HAS_TRAIT(M, TRAIT_ABILITY_BURROWED))
 		return XENO_NO_DELAY_ACTION
 
 	if(HAS_TRAIT(src, TRAIT_ABILITY_BURROWED))
```

Comment:
Line 93 and line 96 are literally the same choose one

Reply:
They're not literally the same, one check is on `M` and one is on `src`. They're two different objects. It could be combined into one if statement but this PR isn't refactoring this proc.